src/luaotfload-auxiliary.lua

```diff
@@ -187,11 +187,10 @@ Comment from fontspec:
 
 local capheight_reference_chars      = { "X", "M", "Ж", "ξ", }
 local capheight_reference_codepoints do
-  local utfbyte = unicode.utf8.byte
   capheight_reference_codepoints = { }
   for i = 1, #capheight_reference_chars do
     local chr = capheight_reference_chars [i]
-    capheight_reference_codepoints [i] = utfbyte (chr)
+    capheight_reference_codepoints [i] = utfcodepoint (chr)
```

Comment:
`utfcodepoint` has exactly the same function as `utfbyte`. It is named differently in `utf8` because in general the result might not actually fit into a byte.